packages/schematics/angular/migrations/migration-collection.json

```diff
@@ -85,15 +85,20 @@
       "factory": "./update-10/update-module-and-target-compiler-options",
       "description": "Update 'module' and 'target' TypeScript compiler options."
     },
-    "update-workspace-dependencies": {
-      "version": "10.0.0-beta.6",
-      "factory": "./update-10/update-dependencies",
-      "description": "Workspace dependencies updates."
-    },
     "update-angular-config": {
       "version": "10.0.0-beta.6",
       "factory": "./update-10/update-angular-config",
       "description": "Remove various deprecated builders options from 'angular.json'."
+    },
+    "update-workspace-dependencies": {
+      "version": "10.0.0-beta.7",
+      "factory": "./update-10/update-dependencies",
+      "description": "Workspace dependencies updates."
+    },
+    "update-libraries-tslib": {
+      "version": "10.0.0-beta.7",
+      "factory": "./update-10/update-libraries-tslib",
+      "description": "Update library projects to use tslib version 2 as a direct dependency."
```

Comment:
If you are a author of a private library and you control all the dependencies, it should be mentioned that you can still use `tslib` as a peer dependency if you use version 2 across the board.

Reply:
I personally don’t think that we should make any distinction between private and public libraries. Since private libraries can also have different private consumers when published.

In case you are using your private library in the same mono repo peerDependencies and dependencies have no effect, because the library is never installed.

Also, since tslib is coupled with the TypeScript version that the library author compiled the library with, it should not be up to the consumer to decide which version of tslib to install, but rather it is the responsible of the library author to choose the correct tslib version. Example if a library was compiled using TS 3.9 it should depend on tslib 2, while if it was compiled using TS 3.8, it should depend upon tslib 1.13.

Reply:
I see. Thanks for the feedback.